src/cryptonote_core/service_node_quorum_cop.cpp

```diff
@@ -375,24 +375,24 @@ namespace service_nodes
 
                   if (info.is_decommissioned()) {
                     if (credit >= 0) {
-                      LOG_PRINT_L2("Decommissioned service node "
+                      LOG_PRINT_L3("Decommissioned service node "
```

Comment:
Debug level (i.e. L2) seems appropriate for these messages.  You don't normally run a service node in debug log level, so I can't see how this change has any practical performance impact.